rollup.config.js

```diff
@@ -1,44 +1,37 @@
-const nodeResolve = require("rollup-plugin-node-resolve");
-const babel = require("rollup-plugin-babel");
-const replace = require("rollup-plugin-replace");
-const uglify = require("rollup-plugin-uglify");
-const fs = require("fs");
+const nodeResolve = require('rollup-plugin-node-resolve');
+const babel = require('rollup-plugin-babel');
+const replace = require('rollup-plugin-replace');
+const { terser } = require('rollup-plugin-terser');
```

Comment:
I know this is a sensitive topic here and I am not forcing this. I just wanted to bring up the suggestion. Terser is quite happily adopted in the community by now. The main reason is that uglify didn't move to support newer language features if I recall.

1. https://github.com/terser/terser
2. https://github.com/mishoo/UglifyJS2

I am fine either way but didn't have any bad experiences with terser.

Reply:
:+1: Fine with trying terser